README.md

```diff
@@ -70,43 +70,77 @@ Examples of `Build` resource using the example strategies shipped with this oper
 ## Try it!
 
 * Get a [Kubernetes](https://kubernetes.io/) cluster and [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) set up to connect to your cluster.
-* Install [Tekton](https://cloud.google.com/tekton) by running [install-tekton.sh](hack/install-tekton.sh), it installs v0.14.2.
-* Install [operator-sdk][operatorsdk] by running [install-operator-sdk.sh](hack/install-operator-sdk.sh), it installs v0.17.0.
-* Create a namespace called **build-examples** by running `kubectl create namespace build-examples`.
-* Execute `make local` to register [well-known build strategies](samples/buildstrategy) including **Kaniko** and start the operator locally.
-* Create a [Kaniko](samples/build/build_kaniko_cr.yaml) build.
+* Create a namespace to run your builds, ex:
+
+```bash
+$ kubectl create namespace build-demo
```

Comment:
I think this is not needed. The quick start example seems to apply all resources in the `default` namespace.